bin/check_samplesheet.py

```diff
@@ -98,7 +98,7 @@ def _validate_pair(self, row):
         if row[self._first_col] and row[self._second_col]:
             row[self._single_col] = False
             assert (
-                Path(row[self._first_col]).suffixes == Path(row[self._second_col]).suffixes
+                Path(row[self._first_col]).suffixes[-2:] == Path(row[self._second_col]).suffixes[-2:]
```

Comment:
@Midnighter any idea what this is doing? :thinking:

Reply:
`suffixes` splits on `.` so if the filename contains dots, there might be unexpected suffixes. We expect the filename to end in `.fastq.gz` so we only make sure that the last two extensions are the same.